src/SpedreadWindow.vala

```diff
@@ -4,6 +4,7 @@ class SpedreadWindow : Gtk.ApplicationWindow {
 
     Gtk.ShortcutController _shortcut_controller;
     Gtk.SpinButton _ms_per_word;
+    static Gtk.SpinButton _words_at_a_time;
```

Comment:
This should not be `static` since it would mean that trying to access `_words_at_a_time` from any window will simply return the last instance created rather than its own instance. Instead, `SpedreadWindow.next_word` should simply be marked non-`static` since it now depends on a property of the current window